### PR TITLE
Incorporate celery_worker into wt stack

### PR DESCRIPTION
**Pull request by Xarthisius**
With https://github.com/whole-tale/gwvolman/pull/199 and https://github.com/whole-tale/girderfs/pull/38 celery container no longer needs to be treated differently and now can be integrated into `docker-stack.yml`

It also requires https://github.com/whole-tale/girder_wholetale/pull/569 even though passthrough is not used yet.

### Note
`docker stack config --compose-file docker-stack.yml` that we're now using to resolve env vars requires fairly new docker-cli. It's not clear to me when they've added it (see https://docs.docker.com/engine/reference/commandline/stack_config/). FWIW I'm using 23.0.4

### CODE REVIEW

File: docker-stack.yml [hunk 1]
```diff
@@ -42,6 +42,8 @@ services:
     volumes:
       - ./volumes:/tmp/data
       - ./girder/girder.local.cfg:/girder/girder/conf/girder.local.cfg
+      - ./src/sem_viewer:/girder/plugins/sem_viewer
+      - ./src/table_view:/girder/plugins/table_view
```
Comment (craig-willis) [inline docker-stack.yml]: Do we really want to add these to deploy-dev?

File: .env [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export docker_group=$(getent group docker | cut -d: -f3)
+export domain=local.wholetale.org
+export registry_user=fido
+export registry_pass=secretpass
+export r2d_version=wholetale/repo2docker_wholetale:latest
```
Comment (craig-willis) [inline .env:5]: Add `export matlab_file_installation_key=secretkey`?

File: docker-stack.yml [hunk 3]
```diff
@@ -169,41 +173,49 @@ services:
...
+      - REGISTRY_USER=${registry_user}
+      - REGISTRY_URL=https://registry.${domain}
+      - REGISTRY_PASS=${registry_pass}
+      - WT_LICENSE_PATH=${PWD}/volumes/licenses
+      - WT_VOLUMES_PATH=${PWD}/volumes
+      - MATLAB_FILE_INSTALLATION_KEY=${matlab_file_installation_key}
+      - SWARM_NODE_ID={{.Node.ID}}
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ./volumes/tmp:/tmp
```
Comment (craig-willis) [inline docker-stack.yml:205]: `volumes/tmp` needed to be to be added to `SUBDIRS` in Makefile for me to ensure creation during `make dev`

File: docker-stack.yml [hunk 4]
```diff
@@ -169,41 +173,49 @@ services:
...
+      - ./src/gwvolman:/gwvolman
+      - ./src/girderfs:/girderfs
+      - type: bind
+        source: ${PWD}/volumes
+        target: ${PWD}/volumes
+        bind:
+          propagation: rshared
```
Comment (craig-willis) [inline docker-stack.yml:212]: Why do we need volumes in gwvolman?

**Review by craig-willis — COMMENTED** [review]
Suggested some minor changes and one question

File: docker-stack.yml (hunk 1 above)
Comment (Xarthisius) [reply docker-stack.yml]: https://github.com/whole-tale/deploy-dev/pull/58/commits/dbf62863ead582dfebdb63c715eebd003ee3d87b

File: docker-stack.yml (hunk 3 above)
Comment (Xarthisius) [reply docker-stack.yml:205]: https://github.com/whole-tale/deploy-dev/pull/58/commits/c52f10a7bd1bbb12832367f47eec933c9db84f03

File: .env (hunk 2 above)
Comment (Xarthisius) [reply .env:5]: https://github.com/whole-tale/deploy-dev/pull/58/commits/bd4adb1c2ef05e02b2812cf1e5a5f85da4d67bf1

File: docker-stack.yml (hunk 4 above)
Comment (Xarthisius) [reply docker-stack.yml:212]: Took a look at the code and RRuns are writing `.docker_stats`, `.stdout` etc directly to run's workspace.

Comment (craig-willis) [reply docker-stack.yml:212]: I see it now.

**Review by craig-willis — APPROVED** [review]

**Pull request merged**